### ISSUE TITLE
Incorrect encoding on xterm and console

### ISSUE THREAD

**Issue by vitko-bg** [Ylianst/MeshCentral#864]
When connecting to a windows 10 machine on bulgarian language using Xterm and Console - results are different. Via console shows all cirillyc characters fine, while via Xterm not. See below.
Using latest 0.4.7-s
Console:
![Capture](https://user-images.githubusercontent.com/59362521/73065991-ca6a3600-3e9c-11ea-8697-b9e74a6b43ef.PNG)

Xterm:
![Capture2](https://user-images.githubusercontent.com/59362521/73065995-cfc78080-3e9c-11ea-9379-314b0a296aac.PNG)

**Comment by krayon007** [comment]
If you type osinfo on the console tab, does it say ConPTY true or false?

**Comment by vitko-bg** [comment]
That is the reply from osinfo:
OS: Microsoft Windows [Version 10.0.18362.592] [ConPTY: YES]

**Comment by krayon007** [comment]
So are you saying if you use the classic terminal, it works, but it does not via xterm?

**Comment by vitko-bg** [comment]
Yes, that is right. via classic terminal - cyrillic is fine, but via xterm - strange symbols ....

**Comment by GusevVictor** [comment]
I can confirm this problem. Cyrlic not working in xterm.js when connect to Windows.

**Comment by Ylianst** [comment]
Published MeshCentral v0.4.8-g with improved XTerm.js support, if you can test this again and let us know if it works now.

**Comment by GusevVictor** [comment]
> 
> 
> Published MeshCentral v0.4.8-g with improved XTerm.js support, if you can test this again and let us know if it works now.

Classic shell works:

![изображение](https://user-images.githubusercontent.com/7081357/73521068-204b5a80-4416-11ea-9ef7-927345b6bd79.png)

Xterm.js not... :(

![изображение](https://user-images.githubusercontent.com/7081357/73521092-335e2a80-4416-11ea-915c-77711c85ec44.png)

**Comment by Ylianst** [comment]
Hi. Can you try this. Go on XTerm and type `chcp 65001`, then try `ipconfig` again and let us know what happens on your system. Thanks.

**Comment by GusevVictor** [comment]
Hi!

![изображение](https://user-images.githubusercontent.com/7081357/73633618-2d638600-4670-11ea-98d1-b6d0e2a44772.png)

**Comment by Ylianst** [comment]
If you just type `chcp`, what does it say? It should give us your codepage. That will be useful so we can test it on our systems. Thanks.

**Comment by GusevVictor** [comment]
> 
> 
> If you just type `chcp`, what does it say? It should give us your codepage. That will be useful so we can test it on our systems. Thanks.

it returns: current code page 0 :(

**Comment by Ylianst** [comment]
@GusevVictor that is interesting.

**Comment by GusevVictor** [comment]
> 
> 
> @GusevVictor that is interesting.

I hope you will hant and fix that. :) I have entered the same command in classic shell with the same result but in a classic shell Cyrlic works fine...
